librz/core/cmd/cmd_debug.c

```diff
@@ -3415,3 +3315,56 @@ RZ_IPI RzCmdStatus rz_cmd_debug_window_identify_handler(RzCore *core, int argc,
 	return RZ_CMD_STATUS_ERROR;
 #endif
 }
+
+RZ_IPI RzCmdStatus rz_cmd_debug_signal_handler(RzCore *core, int argc, const char **argv) {
+	int signum = rz_num_math(core->num, argv[1]);
+	rz_debug_kill(core->dbg, core->dbg->pid, core->dbg->tid, signum);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_signal_list_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	rz_core_debug_signal_print(core->dbg, state);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_signal_name_handler(RzCore *core, int argc, const char **argv) {
+	int signum = rz_num_math(core->num, argv[1]);
+	const char *signame = rz_signal_to_string(signum);
+	if (!signame) {
+		RZ_LOG_ERROR("Invalid signal number\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_cons_println(signame);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_signal_number_handler(RzCore *core, int argc, const char **argv) {
+	int signum = rz_signal_from_string(argv[1]);
+	if (signum <= 0) {
+		RZ_LOG_ERROR("Invalid signal name\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_cons_printf("%d\n", signum);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_debug_signal_option_handler(RzCore *core, int argc, const char **argv) {
+	int signum = rz_num_math(core->num, argv[1]);
+	if (argc < 3) {
+		// Reset the signal
+		rz_debug_signal_setup(core->dbg, signum, 0);
+		return RZ_CMD_STATUS_OK;
+	}
```

Comment:
could you remove this and make the `default_value` of the second argument `reset` so that it will automatically follow that path?